docs/source/howto/workflows.rst

```diff
@@ -18,7 +18,157 @@ Writing workflows
 Running a predefined workflow
 =============================
 
-`#3992`_
+The first step to running a predefined workflow is loading the work function or work chain class that defines the workflow you want to run.
+The recommended method for loading a workflow is using the ``WorkflowFactory``, for example:
+
+.. code-block:: python
+
+    from aiida.plugins import WorkflowFactory
+    add_and_multiply = WorkflowFactory('arithmetic.add_multiply')
+    MultiplyAndAddWorkChain = WorkflowFactory('arithmetic.multiply_add')
+
+This is essentially the same as importing the workflow from its respective module, but using the ``WorkflowFactory`` has the advantage that the entry point will not change when the packages or plugins are reorganised.
+This means your code is less likely to break when updating AiiDA or the plugin that supplies the workflow.
+The input argument of the ``WorkflowFactory`` is the so-called *entry point*.
+For each workflow, the entry point usually starts with the name of the plugin package (here ``arithmetic``), followed by a dot and the string identifying the workflow (here ``multiplyadd``).
```

Comment:
Not something to address now, but a point for the future: this is quite a lot of info, that is maybe necessary, but only tangentially related. In the future, we might want to reduce this a bit perhaps and reference a topic section on entry points and the plugin system.

Reply:
I also just noticed that the *entry point* is used before it is defined due to a restructuring of the text. Let me fix that and maybe slim down this paragraph a bit.